bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/handler/ShellyBaseHandler.java

```diff
@@ -1359,7 +1359,7 @@ public void updateProperties(ShellyDeviceProfile profile, ShellySettingsStatus s
         properties.put(PROPERTY_SERVICE_NAME, config.serviceName);
         String deviceName = getString(profile.settings.name);
         properties.put(PROPERTY_SERVICE_NAME, config.serviceName);
-        properties.put(PROPERTY_DEV_GEN, !profile.isGen2 ? "1" : "2");
+        // properties.put(PROPERTY_DEV_GEN, !profile.isGen2 ? "1" : "2");
```

Comment:
Seems unfinished, did you want to show gen 3 as well?

Reply:
fixed